src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    Machine[] machine = {new Bulldozer(), new Excavator(), new Truck()};
 
+        {
+        for (int i=0; i <= machine.length; i++){
```

Comment:
There will be java.lang.ArrayIndexOutOfBoundsException.
You have to change `int i = 0; i < strings.length; i++` or use forEach (https://www.w3schools.com/java/java_foreach_loop.asp)